### PR TITLE
42 resolve problems on get capabilities

### PR DESCRIPTION
**Pull request by nbrinckm**
This PR solves the 2 issues:
- the problem on fetching the wps capabilities (https://github.com/riesgos/async/issues/42)
- the issue with the references (https://github.com/riesgos/async/issues/41)

It also sets sets a route for download the files in the nginx - for those links that the frontend generates.

Base of this PR is that the wps server didn't properly recognized the settings that are set in the wps init container.
We here touch the web.xml of the wps - which makes the tomcat to reload the wps server configuration. With that in place we update the references in the outputs.
Another important thing is to retry to connect to the wps server - the docker compose internal wps may not be that fast to reliver it all on the same time.

### CODE REVIEW

**Comment by nbrinckm** [comment]
I think the retry is maybe better done by @arnevogt in https://github.com/riesgos/async/pull/59 then I did here. Lets discuss that tomorrow.

File: asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/dummy/AbstractWrapper.kt [hunk 1]
```diff
@@ -329,10 +329,30 @@ abstract class AbstractWrapper(val publisher : PulsarPublisher, val wpsConfigura
         LOGGER.info("Start mapping to wps inputs")
         val wpsInputMapper = InputMapper(getWpsIdentifier())
         val wpsInputs = wpsInputMapper.mapInputs(complexInputs, complexInputsAsValues, complexOutputsAsInputs, literalInputs, bboxInputs)
+        var wpsProcess: WPSProcess? = null
```
Comment (MichaelLangbein) [inline asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/dummy/AbstractWrapper.kt:332]: Does this retry require any more coordination with @arnevogt ? If so, we can talk through that during tomorrow's stand-up. 
Looks fine by me, though!

File: asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/process/wps/WPSClientService.kt [hunk 2]
```diff
@@ -10,7 +10,8 @@ class WPSClientService(val config : WPSConfiguration) {
 
     @Synchronized
     fun establishWPSConnection() : WPSClientSession{
-        val wpsClient = WPSClientSession.getInstance()
+        //val wpsClient = WPSClientSession.getInstance()
+        val wpsClient = WPSClientSession()
```
Comment (MichaelLangbein) [inline asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/process/wps/WPSClientService.kt:14]: Does that mean that `WPSClientSession.getInstance()` had returned a stateful singleton which would always return the same result for all your retries (from the previous commit)? Great thing that that is changed now.

**Review by MichaelLangbein — APPROVED** [review]
Looks perfectly fine to me!
PR approved. Feel free to merge this immediately if you want; Otherwise we can still do any coordination tomorrow after the standup.

File: asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/dummy/AbstractWrapper.kt (hunk 1 above)
Comment (MichaelLangbein) [reply asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/dummy/AbstractWrapper.kt:332]: Oh, never mind @nbrinckm : you already said that you wanted to discuss this with @arnevogt . Taking back my comment.

**Pull request merged**